dashboard/src/features/authn/actions.js

```diff
@@ -0,0 +1,34 @@
+import { unauthedClient } from 'utility/environment'
+import { fetchCoreInfo } from 'features/core/actions'
+
+const setClientToken = (token) => ({type: 'SET_CLIENT_TOKEN', token})
```

Comment:
Is it worth the extra level of indirection here to make an actual action creator function?